### PR TITLE
Add activities feature switches

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml [hunk 1]
```diff
@@ -34,6 +34,12 @@ generic-service:
     FEATURE_EVENT_PRISON_VISIT_REVISED: true
     FEATURE_EVENT_PRISON_VISIT_CANCELLED: true
     FEATURE_EVENT_INCENTIVES_IEP_REVIEW_INSERTED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_CREATED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_AMENDED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_EXTENDED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_AMENDED: true
+    FEATURE_EVENT_ACTIVITIES_PRISONER_ALLOCATED: true
+    FEATURE_EVENT_ACTIVITIES_PRISONER_DEALLOCATED: true
```
Comment (steverendell) [inline helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml:42]: This looks like what the schedule event names will be judging from 'activities.activity-schedule.created'

File: helm_deploy/values-preprod.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@ generic-service:
     API_BASE_URL_MAPPING: https://nomis-sync-prisoner-mapping-preprod.hmpps.service.justice.gov.uk
     API_BASE_URL_VISITS: https://visit-scheduler-preprod.prison.service.justice.gov.uk
     API_BASE_URL_INCENTIVES: https://incentives-api-preprod.hmpps.service.justice.gov.uk
-    API_BASE_URL_ACTIVITIES: https://activities-api-dev.prison.service.justice.gov.uk
     # point at self until real API exists
+    API_BASE_URL_ACTIVITIES: https://prisoner-to-nomis-update-preprod.hmpps.service.justice.gov.uk
```
Comment (steverendell) [inline helm_deploy/values-preprod.yaml:18]: pointing at self seems a bit safer

Comment (mikehalmamoj) [reply helm_deploy/values-preprod.yaml:18]: Does this mean we should turn off the feature switches in preprod?

Comment (steverendell) [reply helm_deploy/values-preprod.yaml:18]: yes that makes sense 👍

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**